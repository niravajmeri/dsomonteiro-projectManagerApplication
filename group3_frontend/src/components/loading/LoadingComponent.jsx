import React, { Component } from 'react';
import './LoadingComponent.css';

class LoadingComponent extends Component {
    render() {
        return (
            <div className="container" >
                <center><div class="loader"></div> </center>
            </div >
        );
    }
}

export default LoadingComponent;