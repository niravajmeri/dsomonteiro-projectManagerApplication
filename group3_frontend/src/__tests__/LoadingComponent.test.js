import React from 'react';
import { shallow } from 'enzyme';
import LoadingComponent from '../components/loading/LoadingComponent';

describe('LoadingComponent', () => {
    it('should render correctly', () => {
        const component = shallow(<LoadingComponent />);
        expect(component).toMatchSnapshot();
    });
});
