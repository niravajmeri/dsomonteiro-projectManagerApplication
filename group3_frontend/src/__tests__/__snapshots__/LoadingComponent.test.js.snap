// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingComponent should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LoadingComponent />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <center>
        <div
                className="loader"
        />
         
</center>,
      "className": "container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="loader"
/>,
          " ",
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "loader",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        " ",
      ],
      "type": "center",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <center>
          <div
                    className="loader"
          />
           
</center>,
        "className": "container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="loader"
/>,
            " ",
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "loader",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          " ",
        ],
        "type": "center",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
